Holtron.Net/Network/Encryption/NetEncryptionAESGCM.cs

```diff
@@ -0,0 +1,82 @@
+using System.Security.Cryptography;
+
+namespace Holtron.Net.Network.Encryption
+{
+    public class NetEncryptionAESGCM : INetEncryption
```

Comment:
`NetEncryptionAESGCM` or `INetEncryption` should implement `IDisposable` since this is working with objects that should be disposed when the instance is released.

Reply:
Implemented IDisposable on INetEncryption.